lnd_test.go

```diff
@@ -970,7 +1060,7 @@ func testUpdateChannelPolicy(net *lntest.NetworkHarness, t *harnessTest) {
 	// internalized this fee update. This shouldn't affect the route that
 	// Alice takes though: we updated the Alice -> Bob channel and she
 	// doesn't pay for transit over that channel as it's direct.
-	payAmt := lnwire.MilliSatoshi(2000)
+	payAmt := btcutil.Amount(2)
```

Comment:
Since it is casted to `int64` below, and interpreted as SAT, I think this is what was originally meant. 

In new commit this is changed to 5 SAT to adhere to the custom min htlc of 5000 mSAT.